src/platforms/javascript/common/errors/index.mdx

```diff
@@ -1,30 +0,0 @@
----
-title: Enrich Event Data
```

Comment:
Is removing this file intentional? From the description I expected the moving of the two files (breadcrumbs-related) but not the deletion of this page about errors and the png image.

Reply:
no no this was a mistake... I just fixed it...thanks :)

Reply:
You're welcome ❤️